ibis/backends/base/file/__init__.py

```diff
@@ -175,6 +115,22 @@ def _list_databases_dirs(self, path=None):
                     tables.append(d.name)
         return tables
 
+    def _list_tables_files(self, path=None):
+        # tables are files in a dir
```

Comment:
Yes, happy to work on typing stuff after the release. But I think we should first the broken mypy errors, since we're not validating types at the moment, and adding types that are not checked doesn't add that much value in my opinion.

Reply:
sure that's why i said followup